source/includes/_sequences.md

```diff
@@ -113,7 +132,7 @@ Explain what the operation does
 
 Parameter    | Type    | Description
 ------------ | ------- | -----------
-owner_id     | path    | Id of the company that creates the code
+owner_id     | path    | UUID of the company that creates the code
```

Comment:
I am not completely sure if we should expose the details of the implementation of the IDs to the user. I forces the users to know one more thing that it is not really needed in order to use the API (unless we are going to have them create the IDs). What do you think?

Reply:
I agree. The concept of "Owner" is not really relevant to the end user as (I would hope) they already know who they are, and the system knows who they are from the authentication tokens which belong to a Company (the owner).

Reply:
Ok, if the owner's identification is available in the tokens, it wouldn't be necessary to have it as a parameter. The user might find it weird to know its own id.